packages/application-shell/src/components/quick-access/quick-access.js

```diff
@@ -356,15 +353,12 @@ export default flowRight(
   injectIntl,
   injectFeatureToggles([
     'pimSearch',
-    'customerGroups',
-    'projectSettings',
-    'developerSettings',
-    'projectSettingsChannels',
```

Comment:
I went ahead and did a little clean up. These feature won't be untoggled and we should clean up our used flags in the MC then too.